src/mr.roboto/src/mr/roboto/subscriber.py

```diff
@@ -248,8 +248,14 @@ def run(self):
             users = " @".join(not_foundation)
             msg = (
                 f"@{users} you need to sign the Plone Contributor "
-                f"Agreement to merge this pull request. \n\n"
+                "Agreement to merge this pull request."
+                "\n\n"
                 f"Learn about the Plone Contributor Agreement: {self.cla_url}"
+                "\n\n"
+                "If you have already signed the agreement, "
+                "please allow a week for your agreement to be processed.\n"
+                "Once it is processed, you will receive an email invitation "
+                "to join the `plone` GitHub organization as a Contributor."
```

Comment:
```suggestion
                "to join the `plone` GitHub organization as a Contributor."
                "If after a week, you have not received an invitation, then "
                f"please contact {self.cla_email}."
```

We also need to define `cla_email` as `agreements@plone.org` wherever `cla_url` is located.

Reply:
Argh, and for `unknown` below, we need to add:

If you have sent in your Plone Contributor Agreement, and received an invitation to join the Plone GitHub organization, but your emails are not known, then you might need to add the email address on your Agreement to your GitHub account.

Reply:
@stevepiercy https://github.com/plone/mr.roboto/commit/ba516be0f173bdadb4736185cb46ca766d3be925